### PR TITLE
Test draw conditions

### PR DESCRIPTION
**Pull request by GasiorMateusz**

### CODE REVIEW

**Comment by GasiorMateusz** [comment]
please @MikolajczykDominika @Kacper-Mila @sandrac91 review my and @kattommy work

**Comment by GasiorMateusz** [comment]
" if there one player chooses the same position on the board 3 times" it is not implemented yet. I am working on it.

**Comment by GasiorMateusz** [comment]
" if there one player chooses the same position on the board 3 times". It is done partially tested, as much as it was possible.

File: src/main/java/draughts/Game.java [hunk 1]
```diff
@@ -35,57 +33,169 @@ public Board getBoard() {
...
+        int moveFactor;
+        Color color;
+        if(player==1){
+            color = Color.WHITE;
+            moveFactor = 1;
+        }else{
+            color = Color.BLACK;
+            moveFactor = -1;
+        }
+        for (int i = 0; i < board.getBoardSize(); i++) {
```
Comment (MikolajczykDominika) [inline src/main/java/draughts/Game.java]: Above you use row/col and now i/j

Comment (GasiorMateusz) [reply src/main/java/draughts/Game.java]: fixed

**Review by sandrac91 — APPROVED** [review]

**Review by MikolajczykDominika — APPROVED** [review]
it lokks quite good, I havent enough time to analyze this changes deeply, I trust you, it work correct and dont crash our program :) so I approve

**Comment by GasiorMateusz** [comment]
I hope 2 approvals are enough :)

**Pull request merged**